### PR TITLE
Rebalancing: removing STUCK from active rses 425

### PR DESCRIPTION
**Pull request by TomasJavurek**
Rebalancing: removing STUCK from active rses 425

### CODE REVIEW

File: lib/rucio/daemons/bb8/t2_background_rebalance.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
 rses_under_ratio = sorted([rse for rse in rses if rse['ratio'] < global_ratio - global_ratio * tolerance], key=lambda k: k['ratio'], reverse=False)
 
 session = get_session()
-active_rses = session.query(models.ReplicationRule.rse_expression).filter(or_(models.ReplicationRule.state == RuleState.REPLICATING, models.ReplicationRule.state == RuleState.STUCK),
+active_rses = session.query(models.ReplicationRule.rse_expression).filter(or_(models.ReplicationRule.state == RuleState.REPLICATING),
```
Comment (bari12) [inline lib/rucio/daemons/bb8/t2_background_rebalance.py:55]: The or_ is a bit redundant now then 😄

**Comment by bari12** [comment]
Pinging @TomasJavurek

**Comment by mlassnig** [comment]
Pinging @TomasJavurek

**Comment by TomasJavurek** [comment]
Can we postpone this a bit? I'll have another version next week. This was just a test.

**Comment by bari12** [comment]
Closing.

**Pull request closed without merging**